lib/mobility/arel/nodes/mysql_ops.rb

```diff
@@ -0,0 +1,50 @@
+# frozen-string-literal: true
+require "mobility/arel"
+
+module Mobility
+  module Arel
+    module Nodes
+      %w[
+        JsonDashArrow
+        JsonDashDoubleArrow
+      ].each do |name|
+        const_set name, (Class.new(Binary) do
+          include ::Arel::Expressions
+          include ::Arel::Predications
+          include ::Arel::OrderPredications
+          include ::Arel::AliasPredication
+
+          def lower
+            super self
+          end
+        end)
+      end
+
+      class Json < JsonDashDoubleArrow; end
+
+      class JsonContainer < Json
+        def initialize column, locale, attr
+          super(Arel::Nodes::JsonDashArrow.new(column, locale), attr)
+        end
+      end
```

Comment:
It's great that you're trying to support a container format for MySQL Json, but I think it will make things a bit complicated for this first PR. How about removing this and leaving it for another PR later?